ISBN/src/d/ISBN.php

```diff
@@ -110,6 +119,13 @@ private function splitIsbn(string $isbn) : array
         return preg_split("/[-\040]+/", $isbn);
     }
 
+    private function ensureRightLength(string $cleanIsbn)
```

Comment:
Ein alphanumerischer ISBN kommt nicht durch die bereits bestehenden validierungen, keine chance. folgende nummern getestet: 978-a-abcdef-abc-d, absdhfkrj, abs-asdfg-as-w, etc. keine schafft es durch die validierungen. ist es in einem solchen fall wirklich notwendig extra noch eine prüfung auf alpanum. zeichen einzubauen?